apps/arweave/src/ar_sync_record.erl

```diff
@@ -52,7 +52,9 @@
 	%% The index of the storage module; undefined for the "default" storage.
 	storage_module_index,
 	%% The number of entries in the write-ahead log.
-	wal
+	wal,
+	%% Whether the sync record is in memory only.
```

Comment:
Is this only true for tests?

Reply:
yes, the new tests in ar_peer_intervals.erl